docs/applications/plex.mdx

```diff
@@ -103,6 +103,109 @@ import SystemdTabs from '../snippets/systemdtabs.mdx';
 
 <SystemdTabs service="plexmediaserver"/>
 
+## Plex Nginx Configuration (optional)
+
+### Subdomain
+
+This method serves Plex on it's own domain. This can be a subdomain you've registered through [LetsEncrypt](https://swizzin.ltd/applications/letsencrypt), answering "no" to applying the config to the swizzin default. After this, you can use the nginx configuration below with some minor modifications in order to get Plex up and serving on your own (sub)domain. The following can either be appended to ``/etc/nginx/sites-enabled/default`` or can (better method) be put into it's own file at ``/etc/nginx/sites-enabled/plex.conf``. Prior to installing this configuration, navagate to [app.plex.tv](https://app.plex.tv) and use the settings listed in the box below. After making changes, ``sudo nginx -t`` to test your config, then ``sudo nginx -s reload`` to apply it. You can also apply ``allowLocalhostOnly="1"`` to your Plex ``Preferences.xml`` file to force it to stop serving on the port.
```

Comment:
Can you please break up this long line into paragraphs or at least with line breaks?